static/app/views/issueList/overviewFc.tsx

```diff
@@ -532,113 +514,99 @@ function IssueListOverviewFc({
 
       setError(null);
 
-      if (lastRequestRef.current) {
-        lastRequestRef.current.cancel();
-      }
-      if (lastStatsRequestRef.current) {
-        lastStatsRequestRef.current.cancel();
-      }
-      if (lastFetchCountsRequestRef.current) {
-        lastFetchCountsRequestRef.current.cancel();
-      }
-
+      api.clear();
       pollerRef.current?.disable();
 
-      lastRequestRef.current = api.request(
-        `/organizations/${organization.slug}/issues/`,
-        {
-          method: 'GET',
-          data: qs.stringify(requestParams),
-          success: (data, _, resp) => {
-            if (!resp) {
-              return;
-            }
-
-            // If this is a direct hit, we redirect to the intended result directly.
-            if (resp.getResponseHeader('X-Sentry-Direct-Hit') === '1') {
-              let redirect: string;
-              if (data[0]?.matchingEventId) {
-                const {id, matchingEventId} = data[0];
-                redirect = `/organizations/${organization.slug}/issues/${id}/events/${matchingEventId}/`;
-              } else {
-                const {id} = data[0];
-                redirect = `/organizations/${organization.slug}/issues/${id}/`;
-              }
+      api.request(`/organizations/${organization.slug}/issues/`, {
```

Comment:
Best to view this diff with "hide whitespace" because there are only a few actual changes here